### DISCUSSION TITLE
Anthropic Claude chatbot support

### DISCUSSION THREAD

**Discussion by antonme** [danny-avila/LibreChat#38] [Feature Requests & Suggestions]
Adding Anthropic Claude bot to the list would be a great idea as it possesses unique strengths and weaknesses compared to ChatGPT or Sydney. Currently, it is only accessible through Quora Poe, but there exists a reverse-engineered API for it at https://github.com/vaibhavk97/Poe

**Comment by danny-avila** [comment]
Great idea, I would love to include as many helpful models as possible. I'll look into this reverse engineered api, maybe i can my own client based on that code!

**Comment by kuangxiaoye** [comment]
I also think that the current project really needs Claude's support, this model has unique characteristics.😄

**Reply by danny-avila** [reply]
one of our contributors just got access and he has expressed that he will add it soon!

**Comment by danny-avila** [comment]
It's now added
